waiter/src/waiter/process_request.clj

```diff
@@ -177,38 +177,20 @@
     (cond-> uri
             (not (str/blank? query-string)) (str "?" query-string))))
 
-(defn- wrap-exception
-  "Includes metadata such as cid and status along with the exception."
-  [message exception instance status headers]
-  (ex-info message
-           (merge (metrics/retrieve-local-stats-for-service (scheduler/instance->service-id instance))
```

Comment:
Did we intentionally avoid merging the local stats for the service?

Reply:
The stats are still included, but it is now handled in `handle-response-error`.

Reply:
My bad, missed the `metrics-map` in `handle-response-error`